packages/angular/projects/clr-angular/src/data/datagrid/datagrid.spec.ts

```diff
@@ -673,6 +718,43 @@ export default function (): void {
         const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
         expect(['1', '1', '2', '4', '3', '9']).toEqual([...rows].map(r => r.textContent));
       });
+
+      it('sorting by column works when pagination is also added', function () {
+        // Instantiate the component only to obtain the test data inside
+        const componentInstance = new SortPaginationTest();
+        const testUsers = componentInstance.users;
+        const sortedTestUsers = [...testUsers].sort((a, b) => a.name.localeCompare(b.name));
+        const reverseSortedTestUsers = [...sortedTestUsers].reverse();
+
+        this.context = this.create(ClrDatagrid, SortPaginationTest);
+        const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
+        expect(rows.length).toEqual(10); // 10 items * 1 property = 10 rows
+
+        rows.forEach((row, index) => {
```

Comment:
This could get really noisy when it fails even if one of them is miss-match.

```js
expect(rows.map(r => r.textContent)).toEqual(testUsers.map(u => u.name))
```
Not the best way to do it but will report something like 
```
// expected 
[A, B, C, D]
// actial
[A, C, B, D]
```
And you will get only one error, not ten.